nodes/BraveSearch/operations/_base.ts

```diff
@@ -7,3 +7,26 @@ export interface BraveSearchOperation {
 	parameters: INodeProperties[];
 	buildQuery(parameters: Record<string, any>): Record<string, any>;
 }
+
+/**
+ * Standard buildQuery implementation that properly handles array parameters and filters empty values
+ * @param query - The query parameters object
+ * @returns Processed query parameters suitable for API consumption
+ */
+export function standardBuildQuery(query: Record<string, any>): Record<string, any> {
+	const { query: q, ...rest } = query;
+	const qs: Record<string, any> = { q };
+
+	for (const [key, value] of Object.entries(rest)) {
+		if (value !== undefined && value !== null && value !== '') {
+			// Handle array parameters (like result_filter) by converting to comma-separated string
+			if (Array.isArray(value)) {
+				qs[key] = value.join(',');
+			} else {
+				qs[key] = value;
+			}
+		}
+	}
+
+	return qs;
+}
```

Comment:
Explicit conversion may not be necessary. If I'm not mistaken, Axios handles this internally (likely via `URLSearchParams`). I just did a quick test, and arrays are already converted to CSV when using `URLSearchParams`:

```js
const arrValues = ["brave","search","api"];
const expectedValue = "values=" + arrValues.join(",");

expectedValue === decodeURIComponent(
    new URLSearchParams({ "values": arrValues }
)); // true
```